i18n/da.yml

```
@@ -99,7 +99,9 @@ warning_message: "WARNING"
 # Logging
 log_application_commands_synced: "Applikationskommandoer synkroniseret med succes."
 log_auto_update_completed: "Automatisk grafopdatering afsluttet. Næste opdatering: {next_update}"
+log_auto_update_completed: "Automatisk opdatering fuldført. Næste opdatering: {next_update}"
```

yamllint

[error] duplication of key "log_auto_update_completed" in mapping

(key-duplicates)